### ISSUE TITLE
Barnarda C Glowing Violet Logs render incorrectly

### ISSUE THREAD

**Issue by MarisaKirisame-ze** [BlesseNtumble/GalaxySpace#681]
**Do not remove this template, without it your issue will not be considered!**

**Please fill in the form below:**
------------------------------------------------------------------------
1. Minecraft version: 1.12.2
2. Galacticraft version: 4.0.5
3. GalaxySpace version: 2.1.3
4. AsmodeusCore version (for 2.0.1 version and above): 1.0.1
5. Side (Single player (SSP), Multiplayer (SMP), or SSP opened to LAN (LAN)): SSP
6. Other add-ons: Pixelmon, Patchouli, More Planets, Zollern Galaxy.
------------------------------------------------------------------------
Description of the issue:
For some reason, Barnarda C Glowing Violet Logs render very strangely and allow me to see through blocks. This does not happen with the other animated blocks in GalaxySpace. However it happens with the other glowing blocks. Please ignore the Pokémon on the right.

------------------------------------------------------------------------
Screenshot/Video:
![image](https://github.com/BlesseNtumble/GalaxySpace/assets/86389939/9fcc2630-9097-4be6-9b35-605f6677a4ca)

------------------------------------------------------------------------
Attached log file (or url on pastebin.com):
I cannot provide a log file.
------------------------------------------------------------------------

**Comment by BlesseNtumble** [comment]
you have optifine?

**Comment by MarisaKirisame-ze** [comment]
Yes, I have OptiFine. This does not happen with the previous GS version.
